src/pages/details.scss

```diff
@@ -0,0 +1,17 @@
+@import "../styles/index.scss";
+
+.p-movieimage-height {
+    min-height: 400px;
```

Comment:
You used REMs in one place and PX here. Keep measurement types in CSS consistent.